tests/conftest.py

```diff
@@ -1771,6 +1771,8 @@ def mcg_obj_fixture(request, *args, **kwargs):
     Returns:
         MCG: An MCG resource
     """
+    if config.ENV_DATA["platform"].lower() == constants.OPENSHIFT_DEDICATED_PLATFORM:
+        return None
```

Comment:
Please provide a warning log message that openshift dedicated is used and the object is not created.

Reply:
Done, Thanks